resources/quality/goofoo/pva/goofoo_near_0.40_pva_draft.inst.cfg

```diff
@@ -0,0 +1,24 @@
+[general]
+version = 4
+name = Draft Quality
+definition = goofoo_near
+
+[metadata]
+setting_version = 17
+type = quality
+quality_type = draft
+material = goofoo_pva
+variant = 0.4mm Nozzle
+
+[values]
+wall_thickness = =line_width*2
+adhesion_type = none
+support_pattern = everywhere
+support_pattern = grid
```

Comment:
Support pattern is mentioned twice. Please remove one of the two.

Reply:
Thank you for your suggestions! I have modified it! Please check it! Thanks!